RocketChat/Rocket.Chat.ReactNative
# [FIX] Non-reply attachments displaying time

## Proposed changes


The presence of the message_link key defines if the attachment is a quote or not for the web client. Implemented this to have the same render requirement on here.

## Issue(s)	

Closes #2901

## How to test or reproduce


1. Open General channel on Open.Rocket.Chat in web client and app.
2. In the web client, the Github messages do not have the time rendered. In the app, the time is rendered. This time is 00:00:00 GMT 1st Jan 1970 for all messages. It is inaccurate and should not be rendered.

## Screenshots
Before: 
![Screenshot_20210212-200448](https://user-images.githubusercontent.com/38764067/107781316-c641dc80-6d6d-11eb-9cf3-f10be6ec7a74.jpg)

After:
![IMG_20210212_200731](https://user-images.githubusercontent.com/38764067/107781844-613ab680-6d6e-11eb-9c4c-46af86a9e199.jpg)

## Types of changes




## Checklist



## Further comments
